src/components/Animal/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./style.css";
+
+export const Animal = ({ id, foto, nazev, nazevLatinsky, chooseAnimal }) => {
+
+
+  return (
+    <div className="animal" onClick={() => chooseAnimal(nazev)}>
```

Comment:
používal bych ID než název na interní reprezentaci stavu (ale pokud víš že nazev je unikátní, tak to samozřejmě bude fungovat)